File: src/main/java/com/example/kyudo_app/domain/model/Place.java
package com.example.kyudo_app.domain.model;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Setter
@Getter
@Entity
@Table(name="Place")
public class Place {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name ="place_id")
    private Integer placeId;
    @Column(name ="place_name")
    private String placeName;
    @Column(name ="address")
    private String placePoint;
    // Placeに関連するゲームマッチ（複数のGameMatch）
    @OneToMany(mappedBy = "place")
    private List<GameMatch> gameMatches;
}

File: src/main/kotlin/com/example/kyudo_app/infrastructure/persistence/Place.kt
package com.example.kyudo_app.infrastructure.persistence

import jakarta.persistence.*
import lombok.Getter
import lombok.Setter

@Setter
@Getter
@Entity
@Table(name = "Place")
class Place {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "place_id")
    private var placeId: Int? = null

    @Column(name = "place_name")
    private var placeName: String? = null

    @Column(name = "address")
    private var placePoint: String? = null

    // Placeに関連するゲームマッチ（複数のGameMatch）
    @OneToMany(mappedBy = "place")
    private val gameMatches: MutableList<GameMatch?>? = null
}
